hw/ip/adc_ctrl/dv/cov/adc_ctrl_cov_unr_excl.el

```diff
@@ -0,0 +1,99 @@
+CHECKSUM: "2208754432 2034735209"
+INSTANCE: tb.dut.u_reg.u_adc_lp_sample_ctl_cdc
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2208754432 2034735209"
+INSTANCE: tb.dut.u_reg.u_adc_sample_ctl_cdc
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2208754432 2034735209"
+INSTANCE: tb.dut.u_reg.u_adc_fsm_rst_cdc
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2208754432 2034735209"
+INSTANCE: tb.dut.u_reg.u_adc_chn0_filter_ctl_0_cdc
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2208754432 2034735209"
+INSTANCE: tb.dut.u_reg.u_adc_chn0_filter_ctl_1_cdc
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2208754432 2034735209"
+INSTANCE: tb.dut.u_reg.u_adc_chn0_filter_ctl_2_cdc
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2208754432 2034735209"
+INSTANCE: tb.dut.u_reg.u_adc_chn0_filter_ctl_3_cdc
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2208754432 2034735209"
+INSTANCE: tb.dut.u_reg.u_adc_chn0_filter_ctl_4_cdc
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2208754432 2034735209"
+INSTANCE: tb.dut.u_reg.u_adc_chn0_filter_ctl_5_cdc
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2208754432 2034735209"
+INSTANCE: tb.dut.u_reg.u_adc_chn0_filter_ctl_6_cdc
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2208754432 2034735209"
+INSTANCE: tb.dut.u_reg.u_adc_chn0_filter_ctl_7_cdc
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2208754432 2034735209"
+INSTANCE: tb.dut.u_reg.u_adc_chn1_filter_ctl_0_cdc
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2208754432 2034735209"
+INSTANCE: tb.dut.u_reg.u_adc_chn1_filter_ctl_1_cdc
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2208754432 2034735209"
+INSTANCE: tb.dut.u_reg.u_adc_chn1_filter_ctl_2_cdc
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2208754432 2034735209"
+INSTANCE: tb.dut.u_reg.u_adc_chn1_filter_ctl_3_cdc
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2208754432 2034735209"
+INSTANCE: tb.dut.u_reg.u_adc_chn1_filter_ctl_4_cdc
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2208754432 2034735209"
+INSTANCE: tb.dut.u_reg.u_adc_chn1_filter_ctl_5_cdc
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2208754432 2034735209"
+INSTANCE: tb.dut.u_reg.u_adc_chn1_filter_ctl_6_cdc
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2208754432 2034735209"
+INSTANCE: tb.dut.u_reg.u_adc_chn1_filter_ctl_7_cdc
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2208754432 2034735209"
+INSTANCE: tb.dut.u_reg.u_adc_chn_val_0_cdc
```

Comment:
these two regs (u_adc_chn_val_0_cdc and u_adc_chn_val_1_cdc): src_busy_q is always 0 because these two registers  src_we_i and src_re_i are tied up to 0.
Reference:
https://github.com/lowRISC/opentitan/blob/master/hw/ip/prim/rtl/prim_reg_cdc.sv#L44
https://github.com/lowRISC/opentitan/blob/master/hw/ip/adc_ctrl/rtl/adc_ctrl_reg_top.sv#L1122